### PR TITLE
docs: add demo gif

### PR DESCRIPTION
**Pull request by ana-oprea**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,13 +14,14 @@
...
 [![Duplicated Lines (%)](https://sonarqube.eea.europa.eu/api/project_badges/measure?project=volto-workflow-progress-develop&metric=duplicated_lines_density)](https://sonarqube.eea.europa.eu/dashboard?id=volto-workflow-progress-develop)
 
-
 Toolbar enhanced workflow [Volto](https://github.com/plone/volto) add-on
 
 ## Features
 
 - Toolbar enhanced workflow slot
 
+![Workflow Progress](./docs/volto-workflow-progress.gif)
```
Comment (avoinea) [inline README.md:23]: @ana-oprea Please update path to absolute url

**Review by avoinea — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ana-oprea) [reply README.md]: @avoinea I updated the path to the gif

**Comment by avoinea** [comment]
@ana-oprea Please fix the merge conflict in README.md

**Comment by ana-oprea** [comment]
@avoinea fixed the merge conflict

**Pull request merged**